Validation/RecoMuon/python/customiseForNewMuonValidation.py

```diff
@@ -0,0 +1,109 @@
+import FWCore.ParameterSet.Config as cms
+
+def enableNewMuonVal(process):
```

Comment:
Indeed having at least a cycle with the 2 validation ("new" and "old") running in parallel is what we discussed and planned to minimize the disruption in the validation activity.